data_steward/utils/participant_summary_requests.py

```diff
@@ -0,0 +1,114 @@
+"""
+Utility to fetch and store deactivated participants information.
+
+The intent of this module is to call and store deactivated participants information by leveraging the RDR Participant
+    Summary API. Deactivated participant information stored is `participantID`, `suspensionStatus`, and `suspensionTime`.
+"""
+
+# Python imports
+import os
+
+# Third party imports
+import google.auth
+import google.auth.transport.requests
+import requests
+import pandas
+from google.oauth2 import service_account
+
+
+def get_access_token(sa_key):
+    """
+    Obtains GCP Bearer token
+
+    :param sa_key: path that points to your cdr-ops SA key
+
+    :return: returns the access_token
+    """
+
+    scopes = [
+        'https://www.googleapis.com/auth/cloud-platform', 'email', 'profile'
+    ]
+
+    service_account_file = sa_key
```

Comment:
What happens if you pass `None` in to the `sa_key`?

Reply:
If there isn't an `sa_key` or if it's `None` then an access token can't be generated. I will add a check here to make sure that one is passed.

Reply:
I thought that might be the case.  I was wondering how we should handle that.  We can explicitly check and raise an error if it isn't given or we can let the software raise the error for us when the `service _account_file` variable is used.  But I don't know what error that would raise.

Reply:
I'll see what error it raises and post it here.

Reply:
Here's the error:

`TypeError: expected str, bytes or os.PathLike object, not NoneType`